### DISCUSSION TITLE
[Help]: Scale Example in Sveltekit?

### DISCUSSION THREAD

**Discussion by XTREEMMAK** [davidjerleke/embla-carousel#807] [Help]
### Summary

Does anyone have a working version of the scale example for Sveltekit?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [X] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Additional information

```js
I would like to get this example working but in Sveltekit:
https://www.embla-carousel.com/examples/predefined/#scale
```

### CodeSandbox example

https://codesandbox.io/p/sandbox/embla-carousel-scale-vanilla-43q6sr?file=%2Fsrc%2Fjs%2FEmblaCarouselTweenScale.js%3A56%2C23

**Comment by davidjerleke** [comment]
- Vote here: #770.
